tests/ui/fmt/help-message-issue-129966-reorder-remove-format-specifier.rs

```diff
@@ -0,0 +1,14 @@
+// Test checks for the help messages in the error output/
+//@ check-run-results
```

Comment:
Problem [REV-RUSTFIX 1/4]: we want to check the suggestion is machine-applicable, so we need to `//@ run-rustfix` instead of `check-run-results` which checks output of test binary. See the file-level comment tagged with REV-RUSTFIX OVERALL about what I think the test should look like.